app/src/main/java/org/simple/clinic/ViewControllerBinding.kt

```diff
@@ -22,12 +22,13 @@ fun <T> bindUiToController(
       .mergeWith(screenDestroys)
       .observeOn(io())
       .compose(controller)
-      .delaySubscription(uiChangeDelay.toMillis(), TimeUnit.MILLISECONDS)
+      .delaySubscription(uiChangeDelay.toMillis(), TimeUnit.MILLISECONDS, mainThread())
```

Comment:
Curious question: what's the use of `delaySubscription()` in this context?

Reply:
That's to subscribe to the view events on the main thread. With the changes to the arch we are running, subscriptions started happening on the UI thread, and RxBinding started complaining and crashing. 💣

Reply:
Got it